src/add-simple-form-value.js

```diff
@@ -21,5 +24,32 @@ export function addSimpleFormValue(value, options) {
     triplesToAdd.push(newTriple);
   }
 
+  // Add missing FormDataNode types for subjects without (rdf:type form:FormDataNode)
+  const triplesToAddLength = triplesToAdd.length;
+
+  for (let index = 0; index < triplesToAddLength; index++) {
+    if (URI_TEMPLATE != "http://data.lblod.info/form-data/nodes/") {
+      console.warn(
```

Comment:
That's not what I meant; I just meant add a comment 'TODO: behavior might need some refinement once we URI patterns are defined in the form'

Reply:
I would remove this logic; what would be the point to use an environment variable then?